IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml.cs

```diff
@@ -56,14 +56,14 @@ public void OnGet()
 
 
                     // grab our JSON text.
-                    string detailBrewery = webClient.DownloadString("https://api.openbrewerydb.org/breweries/10-56-brewing-company-knox");
-                    string detailBreweryresult = "";
+                    string detailBrewery = string.Empty;
+                   
 
 
 
                     try
                     {
-                        detailBreweryresult = webClient.DownloadString(detailBrewery);
+                        detailBrewery = webClient.DownloadString("https://api.openbrewerydb.org/breweries/10-56-brewing-company-knox");

```

Comment:
Added try catch block

Reply:
Logging is good, but what should you do with the exception at this point?  Re-throw it?  If you just log it and bury it, the program will continue to execute as if nothing went wrong.  That might not be the desired behavior.

Use proper logging instead of Console.WriteLine().
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see Console.WriteLine(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().